### ISSUE TITLE
Clearer identification of missing templates

### ISSUE THREAD

**Issue by barnett11** [insightsengineering/chevron#697]
When we run through the workflow, `citril` will create a template program script for a template using `iris` data when the template does not exist in `chevron` - is there a clearer way to identify these to the user? Maybe program script should append a suffix like `_TOUPDATE` or similar so that users know which templates they need to add their own programming to? Seems dangerous to allow an executable script of `iris` data when we know it's incorrect

**Comment by clarkliming** [comment]
yes we can update the dummy template (and probably rename it to something like "error_template“？ because dummy seems like something works
